queries/nickel/highlights.scm

```diff
@@ -0,0 +1,64 @@
+(comment) @comment @spell
+
+[
+  "if"
+  "then"
+  "else"
+  "forall"
```

Comment:
`@repeat`?

Reply:
Not sure, `forall` is used to define polymorphic types: https://nickel-lang.org/user-manual/typing/#type-polymorphism
It's not really a loop